docs/sources/configuration/metrics-config.md

```diff
@@ -39,6 +39,15 @@ define one instance.
 # A value of 0 disables periodic cleanup of abandoned WALs
 [wal_cleanup_period: <duration> | default = "30m"]
 
+# Allows to disable HTTP Keep-Alives when scraping; the Agent will only use
+# outgoing each connection for a single request.
```

Comment:
Something about this verbiage seems off. should `outgoing` and `each` be switched?

Reply:
Yeah sorry, fixed!